common/browser_context.go

```diff
@@ -212,7 +215,7 @@ func (b *BrowserContext) ExposeFunction(name string, callback goja.Callable) {
 }
 
 // GrantPermissions enables the specified permissions, all others will be disabled.
-func (b *BrowserContext) GrantPermissions(permissions []string, opts goja.Value) {
+func (b *BrowserContext) GrantPermissions(permissions []string, parsedOpts *GrantPermissionsOptions) error {
```

Comment:
Should we just say `opts` here as `BrowserContext` now doesn't care about options parsing?

```suggestion
func (b *BrowserContext) GrantPermissions(permissions []string, opts *GrantPermissionsOptions) error {
```